core/frontend/src/AccuDraw.ts

```diff
@@ -1272,18 +1274,54 @@ export class AccuDraw {
     return (!IModelApp.toolAdmin.gridLock);
   }
 
+  /** Call from an AccuDraw UI event to check if key is valid for updating the input field value. */
+  protected itemFieldInputIsValid(key: string, item: ItemField): boolean {
+    if (1 !== key.length)
+      return false;
+
+    switch (key) {
+      case ".":
+      case ",":
+        return true;
+
+      case "'": // feet/minutes...
+      case "\"": // inches/seconds...
+        return true;
+
+      case "°": // degrees...
```

Comment:
Should this switch statement have a case for "^" for degrees? I can't tell.

Reply:
Not necessary, this method is mainly to support NSEW for bearing directions. I don't have an exhaustive list of what every parser might accept and just wanted to call out a few obvious things to early return before the number and letter checks. The "^" -> "°" replacement is only supported in the UI subclass as well...